tests/frontend/grpc/triton_echo_client.py

```
@@ -43,3 +43,17 @@ def run_infer():
 
     assert np.array_equal(input0_data, output0_data)
     assert np.array_equal(input1_data, output1_data)
+
+
+def get_config():
+    server_url = "localhost:8000"
+    try:
+        triton_client = grpcclient.InferenceServerClient(url=server_url)
+    except Exception as e:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)